README.md

```diff
@@ -17,15 +17,80 @@ The latest stable releases on master support the pancancer project https://icgc.
 
 See the Consonance [wiki](https://github.com/Consonance/consonance/wiki) for more information on this project.
 
-## Building
+## Dependencies
+
+I'm showing how to install dependencies on a Mac so adapt to your system/OS.
+
+### PostgreSQL
+
+You need a PostgreSQL server to run the tests, on the Mac try [Postgres.app](http://postgresapp.com/).
+
+Once you have this setup (using whatever technique is appropriate for your system) create the `queue_status` database.
+
+    $bash> "/Applications/Postgres.app/Contents/Versions/10/bin/psql" -p5432 -d "postgres"
```

Comment:
Watch out if unaware, this seems to indicate postgres 10 but the Travis build is running 9.4 which doesn't match. Got bit by this once, took a while to debug.